### PR TITLE
Fixed temporary drawer position

### PR DESCRIPTION
**Pull request by bkarambe**
Fixes # BLUI-3534 #534 

#### Changes proposed in this Pull Request:
- Fixed temporary drawer position in example

#### Screenshots:
Screenshot 2022-10-03 at 10 30 33 PM

#### To Test:
- yarn start:reactdev

### CODE REVIEW

**Comment by huayunh** [comment]
I can't seem to get any temporary drawer to show up... 

https://user-images.githubusercontent.com/8997218/193665220-f0ccfdbb-85d7-4a19-a798-3a1e99025ed0.mov

**Comment by huayunh** [comment]
I can't seem to get the temporary drawer show up at all. 

https://user-images.githubusercontent.com/8997218/193665712-947bcd0c-2e82-46c4-af51-a51dca1443af.mov

**Comment by emclaug2** [comment]
@huayunh @bkarambe Yep same here, the body of the temp drawer is missing.

**Comment by bkarambe** [comment]
> @huayunh @bkarambe Yep same here, the body of the temp drawer is missing.

Seems to be working fine at my local. There might be a build issue, can you please try building components manually and yarn start inside /docs.  This is today's screen recording:

https://user-images.githubusercontent.com/25982779/193762390-1cb8d475-159b-4161-be78-85cd842af122.mov

**Comment by emclaug2** [comment]
> Seems to be working fine at my local. There might be a build issue, can you please try building components manually and yarn start inside /docs. This is today's screen recording:

See here: https://blui-react-docs--pr575-feature-3534-534-tem-3sfz40vy.web.app/components/drawer/examples

**Comment by huayunh** [comment]
I did a fresh clone, and `yarn && yarn start`. I got the same result.

You should ensure that your project will build and run on a fresh clone, without anyone having to manually copy & paste anything. If this means changing the build script, please go ahead and change it.

**Comment by bkarambe** [comment]
> I did a fresh clone, and `yarn && yarn start`. I got the same result.
> 
> You should ensure that your project will build and run on a fresh clone, without anyone having to manually copy & paste anything. If this means changing the build script, please go ahead and change it.

Sorry I forgot to commit library changes. I should be fixed now.

**Comment by emclaug2** [comment]
Hey @bkarambe can you make the Temp Drawer example look like the angular one below:

image

**Comment by emclaug2** [comment]
It looks a lot better, thanks for updating.

But now the code snippet below doesn't match what's being displayed above; it has to include the Toolbar Header & Content Body:

```

            <Box sx={{ backgroundColor: 'background.paper', height: 250 }}>
                <AppBar position="static">
                    <Toolbar>
                        <IconButton
                            size="large"
                            edge="start"
                            color="inherit"
                            aria-label="menu"
                            sx={{ mr: 2 }}
                            onClick={(): void => setOpen(true)}
                        >
                            <Menu />
                        </IconButton>
                        <Typography variant="h6">Toolbar</Typography>
                    </Toolbar>
                </AppBar>
                <Box sx={{ p: 2 }}>App Content Here. </Box>
            </Box>
```

**Comment by huayunh** [comment]
Drawers are floating above the app bar

image

**Comment by bkarambe** [comment]
> Drawers are floating above the app bar
> 
> image

Looks like an issue with dev branch. Logged an issue and will try to fix it in the cleanup story https://github.com/brightlayer-ui/react-component-library/issues/587

**Comment by bkarambe** [comment]
@huayunh @emclaug2 Please review

**Comment by emclaug2** [comment]
> Looks like an issue with dev branch. Logged an issue and will try to fix it in the cleanup story #587

I don't understand why you're saying this is an issue with the dev branch;  I cannot recreate the issue on either the dev deployment here: https://brightlayer-ui-components.github.io/react-dev/components/drawer/examples or whenever I cloned dev & ran it locally.

**Comment by bkarambe** [comment]
> > Looks like an issue with dev branch. Logged an issue and will try to fix it in the cleanup story #587
> 
> I don't understand why you're saying this is an issue with the dev branch; I cannot recreate the issue on either the dev deployment here: https://brightlayer-ui-components.github.io/react-dev/components/drawer/examples or whenever I cloned dev & ran it locally.

Sorry about that, I think that's due to cache issue. Recent commit fixes that.

File: components/src/core/Drawer/Drawer.tsx [hunk 1]
```diff
@@ -110,6 +110,7 @@ const Root = styled(MUIDrawer, {
         position: 'relative',
         boxShadow: sideBorder ? 'none' : theme.shadows[4],
         borderWidth: sideBorder ? 1 : 0,
+        zIndex: 'auto',
```
Comment (emclaug2) [inline components/src/core/Drawer/Drawer.tsx:113]: If the solution was already working on dev before, updating the component library styles should not be required.   The fix should be scoped to the demo application, not our component library.

**Comment by huayunh** [comment]
Drawer behavior looks good!

If these lines below are here so that this temporary drawer can render properly on dev doc, then I recommend taking these out. Our users should not be using them in their application

image

**Review by emclaug2 — APPROVED** [review]
Looks good here, just see Huayun's comment

**Comment by bkarambe** [comment]
> Drawer behavior looks good!
> 
> If these lines below are here so that this temporary drawer can render properly on dev doc, then I recommend taking these out. Our users should not be using them in their application
> 
> image

Updated

**Pull request merged**